surveyform.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SURVEY-FORM</title>
+   <style>.heading{
+    color:purple;
+    text-align: center;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+    font-size: 2em;;
+    font-weight: 1000;
+    font-style: italic;
+    text-decoration:darkmagenta;
+    text-shadow:green;
+    text-transform: uppercase;
+    
+}
+body{
+    background-color:cyan;
+}
+#blue{
+    color:blue;
+}
+    
+form{
+    
+    border:dashed black;
+    margin:auto;
+    border-radius: 10px;
+    color:blue;
+}
+</style>
+</head>
+<body>
+<h1 class="heading"><em>WELCOME TO THE SHORT SURVEY</em></h1>
+   <div>
+     <h2 id="blue">About yourself</h2>
+    <form>
+    <label for="uname">Enter your name:</label>
+    <input type="text" name="uname" placeholder="Enter your name">
+    <label>Gender</label>
+    <input type="radio" name="gender" >Male
+    <input type="radio" name="gender" >Female
+    <label >Locality</label>
+    <input type="text" placeholder="enter the name of your locality" >
+    <label>Qualification</label>
+    <select name="qual">
+        <option>select your qualification</option>
+        <option>B.E</option>
+        <option>B.Tech</option>
+        <option>B.Sc</option>
+        <option>B.Com</option>
+        <option>BCA</option>
+        <option>MBA</option>
+    </select> 
+    <label>Occupation</label>
+    <select name="occupation">
+    <option>select your Occupation</option>
+    <option>Student</option>
+    <option>Faculty</option>
+    <option>Other</option>
+    </select> <br><br>
+  </div>
```

Comment:
You have closed `</div>` but you haven't closed `</form>`.

You have to use HTML this way:

    <a><b></b></a>

Not this way:

    <a><b></a></b>